docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -0,0 +1,641 @@
+Cluster**. Confirm the deletion by entering the cluster name when prompted.
+
+![Delete Palette cluster](/clusters_pcg_deploy-app-pcg_cluster-delete.png)
+
+The cluster status will change to **Deleting**, and the deletion process may take up to 15 minutes.
+
+Once the cluster is deleted, proceed to delete the cluster profile. On the left **Main Menu**, click on **Profiles**,
+and select the profile `pcg-tutorial-profile`. Next, click on the **three-dot Menu**, select **Delete**, and confirm the
+selection to remove the cluster profile.
+
+</TabItem>
+
+<TabItem label="Terraform Workflow" value="Terraform_Workflow">
+
+In your terminal, issue the `terraform destroy` command within the **/terraform/vmware-cluster-deployment-tf** directory
+to remove all the resources that were created by Terraform.
+
+```bash
+terraform destroy --auto-approve
+```
+
+Deleting the resources may take up to 15 minutes. Upon completion, you should receive an output similar to the provided
+below.
+
+```text hideClipboard
+# Output condensed for readability
+Destroy complete! Resources: 5 destroyed.
+```
+
+</TabItem>
+
+</Tabs>
+
+### Delete the PCG
+
+After deleting your VMware cluster and cluster profile, proceed with the PCG deletion. Log in to Palette as a tenant
+admin, navigate to the left **Main Menu** and select **Tenant Settings**. Next, from the **Tenant Settings Menu**, click
+on **Private Cloud Gateways**. Identify the PCG you want to delete, click on the **Three-Dot Menu** at the end of the
+PCG row, and select **Delete**. Click **OK** to confirm the PCG deletion.
+
+![Delete PCG image](/clusters_pcg_deploy-app-pcg_pcg-delete.png)
+
+Palette will delete the PCG and the Palette services deployed on the PCG node. However, the underlying infrastructure
+resources, such as the virtual machine, must be removed manually from VMware vSphere.
+
+Log in to your VMware vSphere server and select the VM representing the PCG node named `gateway-tutorial-cp`. Click on
+the **Three-Dot Actions** button, select **Power**, and **Power Off** to power off the machine. Once the machine is
+powered off, click on the **Three-Dot Actions** button again and select **Delete from Disk** to remove the machine from
+your VMware vSphere environment.
+
+![Delete VMware VM](/clusters_pcg_deploy-app-pcg_vmware-delete.png)
+
+## Wrap-Up
+
+In this tutorial, you learned how to deploy a VMware PCG using Palette CLI and utilize it to deploy a Palette cluster
+with a sample Kubernetes application to your VMware vSphere environment. Next, you accessed the Hello Universe
+application landing page through the exposed service URL.
+
+A PCG is a powerful component that enables Palette to communicate with private clouds or data center environments that
+impose restrictions on external connections. With a PCG, you can deploy Palette clusters in environments such as VMware
+vSphere, while also providing lifecycle support and monitoring for your clusters.
```

Comment:
We should reinforce at the end that the PCG supports using Palette in restricted networks that don't allow inbound connectives from the outside. The PCG acts as the endpoint due to it's design as the source and polls Palette for instructions.  This is how we uses can use Palette without having to worry about making scary firewall rules or exposes public endpoints.

Reply:
I rewrote the paragraph :)